wpeverest/everest-forms
# Tweak - Removed form builder footer

### All Submissions:






### Changes proposed in this Pull Request:



Removed `wpfooter` on form builder page.

### How to test the changes in this Pull Request:

1. Create a new form.
2. Check if there is footer text or version update on the bottom of the page.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Removed form builder footer
